common/src/main/java/gov/cms/ab2d/common/repository/CoverageServiceRepository.java

```diff
@@ -43,6 +43,13 @@
     private static final String SELECT_DISTINCT_COVERAGE_BY_PERIOD_COUNT = "SELECT COUNT(DISTINCT beneficiary_id) FROM coverage" +
             " WHERE bene_coverage_period_id IN(:ids)";
 
+    static final String SELECT_DELTA =
+            "SELECT cov1.bene_coverage_period_id, cov1.beneficiary_id, :type as entryType, CURRENT_TIMESTAMP as created" +
+            " FROM coverage cov1" +
+            " WHERE cov1.bene_coverage_search_event_id = :search1 AND NOT EXISTS" +
+                " (SELECT cov2.beneficiary_id FROM coverage cov2" +
+                " WHERE cov1.beneficiary_id = cov2.beneficiary_id and bene_coverage_search_event_id = :search2 )";
```

Comment:
Why is this not in the DeltaRepository?

Reply:
Because that is a coverage query, not a coverage delta query.  I like to keep all queries specific to a table in the repository that manages that table.